compiler/util/misc.cpp

```diff
@@ -35,6 +35,9 @@
 #include <fcntl.h>
 #include <sys/stat.h>
 
+
+static const char* help_url = "http://chapel.cray.com";
+
```

Comment:
In my mind, ideally we'd have a bugs.html that this would link to. I don't really care right now which URL we use as long as it points there somehow...